slick-testkit/src/main/scala/com/typesafe/slick/testkit/tests/InsertTest.scala

```diff
@@ -196,11 +196,12 @@ class InsertTest extends AsyncTest[JdbcTestDB] {
     val ts = TableQuery[T]
 
     for {
-      _ <- ts.schema.create
-      _ <- ts ++= Seq((1, "a"), (2, "b"))
-      _ <- ts.insertOrUpdate((3, "c")).map(_ shouldBe 1)
-      _ <- ts.insertOrUpdate((1, "d")).map(_ shouldBe 1)
-      _ <- ts.sortBy(_.id).result.map(_ shouldBe Seq((1, "d"), (2, "b"), (3, "c")))
+      // FIXME using underscores here causes a compile error, is this a scala 2.13.0-RC1 bug?
```

Comment:
@szeiger Do you have any idea if this is perhaps a scala 2.13 bug? It seems that the old version of this code does not compile on 2.13.0-RC1